crunchers/cruncher_active_users.h

```diff
@@ -77,6 +74,13 @@ struct ActiveUsersCruncherImpl {
   using user_list_t = std::list<ActiveUser>;
   using user_map_t = std::unordered_map<std::string, typename user_list_t::iterator>;
 
+  void RemoveUsersOutsideTheTimeWindow() {
+    while (!users_list_.empty() && users_list_.back().us + interval_ <= current_us_) {
```

Comment:
It's the same example as the previous one with seconds. If we publish 1 event per microsecond, we expect to see only one event in 1-microsecond window, right?